.github/workflows/release-azure-pipelines.yml

```diff
@@ -93,40 +93,268 @@ extends:
+          - checkout: self
+          - task: UseDotNet@2
+            displayName: Use .NET Core sdk 6.x
+            inputs:
+              version: 6.x
+          - task: NuGetToolInstaller@0
+            displayName: Use NuGet 6.x
+            inputs:
+              versionSpec: 6.x
+          - task: DotNetCoreCLI@2
+            displayName: Install dependencies
+            inputs:
+              command: restore
+              feedsToUse: select
+              vstsFeed: $(vstsFeedId)
+              includeNuGetOrg: false
+              arguments: --runtime ${{ config.runtime }}
+          - task: DotNetCoreCLI@2
+            displayName: Test
+            inputs:
+              command: test
+              arguments: --configuration release --no-restore
+          
+          - task: DotNetCoreCLI@2
+            displayName: Build artifacts
+            env:
+              ADO_TOKEN: $(System.AccessToken)
+            inputs:
+              command: publish
+              projects: src/AzureAuth/AzureAuth.csproj
+              arguments: -p:Version=${{ parameters.version }} --configuration release --self-contained true --runtime ${{ config.runtime }} --output dist/${{ config.runtime }}
+              publishWebProjects: false
+              zipAfterPublish: false
+              modifyOutputPath: true
 
-            - task: NuGetToolInstaller@0
-              displayName: Use NuGet 6.x
-              inputs:
-                versionSpec: 6.x
+    - stage: sign
+      displayName: Sign
+      dependsOn: build
+      jobs:
+      - ${{ each config in parameters.buildConfigs }}:
+        - job: sign_${{ replace(config.runtime,'-', '_') }}
+          displayName: Signing ${{ config.runtime }}
+          pool:
+            name: Azure-Pipelines-1ESPT-ExDShared
+            image: windows-latest
+            os: windows
+          templateContext:
+            inputs:
+            - input: pipelineArtifact
+              artifactName: azureauth-${{ parameters.version }}-${{ config.runtime }}
+              targetPath: $(Build.ArtifactStagingDirectory)/azureauth-${{ parameters.version }}-${{ config.runtime }}
+            outputs:
+            - output: pipelineArtifact
+              artifactName: azureauth-${{ parameters.version }}-${{ config.runtime }}-signed
+              targetPath: $(Build.ArtifactStagingDirectory)/azureauth-${{ parameters.version }}-${{ config.runtime }}-signed
+          steps:
+          - task: EsrpCodeSigning@5
+            displayName: Sign artifacts win10-x64
+            condition: eq('${{ config.runtime }}', 'win10-x64')
+            inputs:
+              ConnectedServiceName: $(esrpKVServiceConnection)
+              AppRegistrationClientId: $(SIGNING_AAD_ID)
+              AppRegistrationTenantId: $(SIGNING_TENANT_ID)
+              AuthAKVName: $(AZURE_VAULT)
+              AuthCertName: $(AZURE_VAULT_ESRP_AAD_CERT_NAME)
+              AuthSignCertName: $(AZURE_VAULT_ESRP_REQ_CERT_NAME)

```

Comment:
Just confirming, but are all of these going to be Pipeline variables? If so, should we use consistent casing for variable names? (I'm not picky on which casing style we choose, so long as we stay consistent)

Reply:
Done!